app/models/katello/glue/candlepin/product.rb

```diff
@@ -70,7 +70,7 @@ def orphaned?
       end
 
       def build_product_content(attrs)
-        @productContent = attrs.collect { |pc| Candlepin::ProductContent.new pc }
+        @productContent = attrs.collect { |pc| ::Katello::Candlepin::ProductContent.new pc }
       end
```

Comment:
Though the leadning `::` isn't incorrect, I believe it's unnecessary in this case.